### DISCUSSION TITLE
Ability to hide sidebar and table of contents

### DISCUSSION THREAD

**Discussion by xperiandri** [requarks/wiki#4891] [General]
Sidebar and table of contents take almost the half of the screen and both are on the right.
I think they must be collapsable (for example like in Azure DevOps)
Or they must go down if you enlarge scaling (like on Microsoft Docs website).

Being able to move a table of contents to the left would also be nice.

@NGPixel

**Comment by fumbles** [comment]
The TOC bar can be hidden in 2.5.297 and higher. 
Under Administration -> Theme

**Reply by joeldeteves** [reply]
> The TOC bar can be hidden in 2.5.297 and higher. Under Administration -> Theme

Yes however making it collapsible and making it hidden are two different things. The sidebar should be collapsible by the user so they can pull out the table of contents and hide it when they like. So I agree with @xperiandri on this

**Reply by fnafox** [reply]
It can be done with some custom code, there's an already existing project that helps with this which you might want to look into, but i agree having support for this natively would be nice
https://github.com/madodig/wikijs-customization

**Reply by nmm2016** [reply]
> 它可以通过一些自定义代码来完成，有一个已经存在的项目可以帮助解决这个问题，你可能想要研究一下，但我同意原生支持它会很好 https://github.com/madodig/wikijs-customization

I just want a native folding button, but it only appears in the bottom left corner when the webpage is zoomed in to a certain extent. Sincerely ask how to modify it

**Reply by pat0s** [reply]
@xperiandri @NGPixel (sorry for the tags guys) 
Is there any update on this feature?

**Comment by eswenson1** [comment]
I'd like to add my thumbs up on this request. The TOC panel should be collapsible by the user -- not through administrator configuration, which impacts all users.  The TOC panel takes up a large part of the screen real estate, and sometimes can be deemed unnecessary distraction by users.  There should be an expand/collapse icon so the panel can be collapsed and expanded at will by any user, logged in or not.

**Comment by ajeanmahoney** [comment]
Also +1
